src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_MAX = 100;
+    private static final int RANDOM_MIN = 1;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random;
+
+    public Lottery() {
+        this.random = new Random();
+    }
+
+    public int getRandomNumber() {
+        return random.nextInt(RANDOM_MAX - RANDOM_MIN + 1) + RANDOM_MIN;
+    }
```

Comment:
this method seems to not be used anywhere except your getRandomBall() method, so let's make it private and place this method after your public one

Reply:
done)